ruff.toml

```diff
@@ -0,0 +1,59 @@
+# Enable the pycodestyle (`E`) and Pyflakes (`F`) rules by default.
```

Comment:
Instead of ruff.toml, could we go with a pyproject.toml that we can share with other linters like black/blue and mypy?